cmd/topicmappr/commands/rebuild.go

```diff
@@ -43,10 +43,9 @@ func init() {
 	rebuildCmd.Flags().Bool("skip-no-ops", false, "Skip no-op partition assigments")
 	rebuildCmd.Flags().Bool("optimize-leadership", false, "Rebalance all broker leader/follower ratios")
 	rebuildCmd.Flags().Bool("phased-reassignment", false, "Create two-phase output maps")
-
-	// new params for evac leadership
 	rebuildCmd.Flags().String("leader-evac-brokers", "", "Broker list to remove leadership for topics in leader-evac-topics.")
 	rebuildCmd.Flags().String("leader-evac-topics", "", "Topics list to remove leadership for the brokers given in leader-evac-brokers")
+	rebuildCmd.Flags().Int("chunk-step-size", 0, "Number of brokers to move data at a time for with a getPartitionMapChunk operation.")
```

Comment:
Might want to avoid using an internal function reference since a user should know what a flag means without necessarily knowing anything about the code

Reply:
Meant that to say 'chunked', was the original name of the function and a victim of the name refactor by goland